internal/scheduler/scheduling_algo.go

```diff
@@ -199,6 +199,10 @@ func (l *LegacySchedulingAlgo) scheduleOnExecutor(
 		nodeDb,
 		priorityFactorByQueue,
 		allocatedByQueueAndPriority,
+		// TODO: Add missing maps to enable gang preemption.
```

Comment:
I wanted to test that the gang preemption design works out before adding it also to the new scheduler.